vignettes/r-notebook.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "R Notebook"
+author: "Kevin Ushey"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{R Notebook}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+This vignette describes the HTML notebook format, and is primarily intended for
+front-end applications using or embedding R, or other users who are interested
+in understanding and parsing the R Notebook format.
+
+R Notebooks are HTML documents with data written and encoded in such a way that:
+
+1. The generating `.Rmd` document can be recovered, and
+2. Chunk outputs can be recovered.
+
+To generate an R Notebook, you can use `rmarkdown::render()` and specify the
+`rmarkdown::html_notebook` output format in your document's YAML metadata.
```

Comment:
`html_notebook` (we automatically default to rmarkdown as the containing package)